wowup-electron/src/app/pages/my-addons/my-addons.component.html

```diff
@@ -1,17 +1,25 @@
 <div class="tab-container d-flex flex-col" [ngClass]="{ mac: electronService.isMac, windows: electronService.isWin }">
   <div class="control-container">
-    <div class="select-container">
-      <mat-form-field>
-        <mat-label i18n="World of Warcraft client selection label">World of Warcraft</mat-label>
-        <mat-select class="select" [(value)]="selectedClient" (selectionChange)="onClientChange()"
-          [disabled]="enableControls === false">
-          <mat-option [value]="clientType" *ngFor="
-              let clientType of warcraftService.installedClientTypes$ | async
-            ">
-            {{ warcraftService.getClientDisplayName(clientType) }}
-          </mat-option>
-        </mat-select>
-      </mat-form-field>
+    <div class="left-container">
+      <div class="select-container">
+        <mat-form-field>
+          <mat-label i18n="World of Warcraft client selection label">World of Warcraft</mat-label>
+          <mat-select class="select" [(value)]="selectedClient" (selectionChange)="onClientChange()"
+            [disabled]="enableControls === false">
+            <mat-option [value]="clientType" *ngFor="
+                let clientType of warcraftService.installedClientTypes$ | async
+              ">
+              {{ warcraftService.getClientDisplayName(clientType) }}
+            </mat-option>
+          </mat-select>
+        </mat-form-field>
+      </div>
+      <div class="filter-container" *ngIf="selectedClient !== wowClientType.None">
```

Comment:
Can we move this to the right side of the bar to be like the windows app?
Can it look like the search bar in the Get Addons tab, with the little x to clear as well.

Reply:
Moved both the filter and search boxes to the left side of the buttons. Also matched the filter box markup with the search box.